wowm_language/src/docs/smsg_compressed_update_object.md

```diff
@@ -23,5 +25,120 @@ SMSG have a header of 4 bytes.
 
 ### Body
 
-The body for this message has not been implemented yet.
+| Offset | Size / Endianness | Type | Name | Description | Comment |
+| ------ | ----------------- | ---- | ---- | ----------- | ------- |
+| 0x04 | 4 / Little | u32 | amount_of_objects |  |  |
+| 0x08 | 1 / - | u8 | has_transport |  |  |
+| 0x09 | ? / - | [Object](object.md)[amount_of_objects] | objects |  |  |
 
+### Examples
```

Comment:
I'm unsure of how to deal with the examples here. On one hand it's useful to see the uncompressed data explained but on the other it gives an incorrect view of what's actually going on.

I also noticed that the example doesn't have the uncompressed size `u32`. Maybe generate example text for both the compressed and uncompressed data?

Reply:
Examples are displayed compressed with u32 for size, and then manually linked to uncompressed version in the comments as described in the other conversation.